fs/config/configflags/configflags.go

```diff
@@ -149,6 +151,19 @@ func SetFlags() {
 		log.Fatalf(`Can only use --suffix with --backup-dir.`)
 	}
 
+	switch {
+		case len(fs.Config.StatsOneLineDateFormat) > 0:
+			fs.Config.StatsOneLineDate = true
+			fs.Config.StatsOneLine = true
+			fs.Config.Progress = true
```

Comment:
I don't think any of these options should imply --progress.  `--stats-one-line` is useful without `--progress`.

Reply:
I'm sorry i'm missing this. I must be understanding the heirarchy wrong.

Base is `--progress`. This can be overridden by `--stats-one-line`, which according to docs `This can be used with the --stats-one-line flag for a simpler display.`

What does  `--stats-one-line` display without `--progress`?